misc/dbt-materialize/dbt/include/materialize/macros/columns.sql

```diff
@@ -0,0 +1,23 @@
+-- Copyright Materialize, Inc. and contributors. All rights reserved.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License in the LICENSE file at the
+-- root of this repository, or online at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+{% macro materialize__get_empty_subquery_sql(select_sql, select_sql_header=none) %}
+    {%- if select_sql_header is not none -%}
+    {{ select_sql_header + '#__dbt_sbq_parse_header__#' }}
```

Comment:
It doesn't matter at all but since this is all Jinja I think you could just instead do:

```suggestion
    {{ select_sql_header }}
    #__dbt_sbq_parse_header__#
```

Reply:
See also comment above about hoisting the `#__dbt_sbq_parse_header__#` partition out of the if, so that it's present regardless of whether the header is.